.circleci/config.yml

```diff
@@ -34,13 +68,21 @@ jobs:
             mv ../*.deb /tmp/deb
       - deploy:
           command: |
-              if [ "${CIRCLE_BRANCH}" == "master" ]; then
-                gem install rake package_cloud
-                package_cloud push wott/agent/debian/stretch /tmp/deb/*.deb
-                package_cloud push wott/agent/raspbian/stretch /tmp/deb/*.deb
-              fi
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              gem install rake package_cloud
+              package_cloud push wott/agent/debian/stretch /tmp/deb/*.deb
+              package_cloud push wott/agent/raspbian/stretch /tmp/deb/*.deb
+            fi
       - store_artifacts:
           path: test-reports
           destination: test-reports
       - store_artifacts:
           path: /tmp/deb
+workflows:
+  version: 2
+  build_all:
+    jobs:
+      - build_debian_buster
+      - build_debian_stretch:
+          requires:
+            - build_debian_buster
```

Comment:
Why do we need to wait here?

Reply:
Just to avoid displaying 'NOT RUNNING' jobs' statuses in the CircleCI UI.

Reply:
I guess it doesn't matter about parallel mode right now, but it will single thread it later.

Reply:
> but it will single thread it later

it's not clear from your comment is this ^ a desired or not.

Reply:
It's fine for now. Let's remove it when we add more.